articles/language/type-model.md

```diff
@@ -14,9 +14,9 @@ uid: microsoft.quantum.language.type-model
 This section lays out the Q# type model and describes the syntax for
 specifying and working with types.
 
-## Primitive Types
```

Comment:
Keeping primitive here is fine, since this usage here has nothing to do with the libraries or the namespace Microsoft.Quantum.Primitive.